src/main/java/cs2103/v15_1j/jimjim/AddTaskCommand.java

```diff
@@ -13,18 +13,24 @@ public AddTaskCommand(String name, LocalDateTime datetime) {
 
 	@Override
 	public String undo(List<TaskEvent> displayList, List<TaskEvent> list, Storage storage, Searcher searcher) {
-	    //TODO
+		// TODO
 		return null;
 	}
 
 	@Override
 	public String execute(List<TaskEvent> displayList, List<TaskEvent> list, Storage storage, Searcher searcher) {
+		// Add to display list first to make it seem more responsive
```

Comment:
Our current program is single threaded and synchronous though, so the UI will not receive the new update until everything is done